### PR TITLE
buildcache: Add version v0.28.4

### LINKED ISSUE #11627: [Request]: buildcache
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

buildcache

### Description

BuildCache is an advanced compiler accelerator that caches and reuses build results to avoid unnecessary re-compilations, and thereby speeding up the build process.

### Homepage

https://github.com/mbitsnbites/buildcache

### Download Link(s)

https://github.com/mbitsnbites/buildcache/releases/download/v0.28.4/buildcache-linux.tar.gz https://github.com/mbitsnbites/buildcache/releases/download/v0.28.4/buildcache-macos.zip https://github.com/mbitsnbites/buildcache/releases/download/v0.28.4/buildcache-windows.zip

### Some Indication of Popularity/Repute

https://github.com/mbitsnbites/buildcache 206 stars and 44 forks

### PR DESCRIPTION
**Pull request by szymonj99**
Closes #11627 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/buildcache.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "version": "0.28.4",
+    "description": "BuildCache is an advanced compiler accelerator that caches and reuses build results to avoid unnecessary re-compilations, and thereby speeding up the build process.",
+    "homepage": "https://github.com/mbitsnbites/buildcache",
+    "license": "Zlb",
```
Comment (HUMORCE) [inline bucket/buildcache.json:5]:
```suggestion
    "license": "Zlib",
```

correct short identifier

File: bucket/buildcache.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/mbitsnbites/buildcache/releases/download/v0.28.4/buildcache-windows.zip",
+            "hash": "8003f9c79152092ec37397873d668d68f6e4cb78ca1ef04389c3f5ad6bac7fc4"
+        },
+		"32bit": {
+            "url": "https://github.com/mbitsnbites/buildcache/releases/download/v0.28.4/buildcache-windows.zip",
+            "hash": "8003f9c79152092ec37397873d668d68f6e4cb78ca1ef04389c3f5ad6bac7fc4"
+        }
```
Comment (HUMORCE) [inline bucket/buildcache.json:14]:
```suggestion
        }
```

64bit program

File: bucket/buildcache.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        "github": "https://github.com/mbitsnbites/buildcache"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/mbitsnbites/buildcache/releases/download/v$version/buildcache-windows.zip"
+            },
+			"32bit": {
+                "url": "https://github.com/mbitsnbites/buildcache/releases/download/v$version/buildcache-windows.zip"
+            }
```
Comment (HUMORCE) [inline bucket/buildcache.json:34]:
```suggestion
            }
```

**Review by HUMORCE — CHANGES_REQUESTED** [review]

File: bucket/buildcache.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    "bin": "bin/buildcache.exe",
+    "shortcuts": [
+        [
+            "bin/buildcache.exe",
+            "buildcache"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/mbitsnbites/buildcache"
+    },
```
Comment (HUMORCE) [inline bucket/buildcache.json:26]:
```suggestion
    "checkver": “github”,
```

**Comment by szymonj99** [comment]
Issues should be fixed (unless I messed up 😄 ). Thank you!

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**